src/components/Search/CategoryFilter.tsx

```diff
@@ -63,6 +63,7 @@ export default React.memo(function CategoryFilter({
       )}
       {filters.map((value, index) => {
         const label = toSnakeCase(value)
+        const isGrant = value == ProposalType.Grant
```

Comment:
Shouldn't you use `isGrantType` for this? 

I'm thinking that if the answer is no and that function is for something else (more specific), it could have a more specific name. Like `isGrantSubtype` maybe?